src/subcmd/withdraw_fees.rs

```diff
@@ -0,0 +1,68 @@
+use base64::{engine::general_purpose, Engine};
+use borsh::ser::BorshSerialize;
+use clap::Args;
+use solana_sdk::pubkey::Pubkey;
+use spl_governance::state::proposal_transaction::{AccountMetaData, InstructionData};
+use stakedex_interface::{WithdrawFeesIxArgs, WithdrawFeesKeys};
+use stakedex_sdk_common::find_fee_token_acc;
+
+use super::SubcmdExec;
+
+#[derive(Args, Debug)]
+#[command(
+    long_about = "Outputs a base64-encoded transaction that withdraws fees to the specified token account when signed by the admin authority."
```

Comment:
nit: my bad, forgot how the governance program worked when i wrote this

```suggestion
    long_about = "Outputs a base64-encoded spl-governance program instruction that withdraws fees to the specified token account when executed by the admin authority spl-governance PDA."
```